content/docs/alerting/webhook_receivers.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Webhook Receivers
+sort_rank: 6
+nav_icon: sliders
+---
+
+# Webhook Receivers
+
+There are a number of 3rd party systems that directly support the Alertmanager webhook receiver.
+
+* [IRC Bot](https://github.com/metalmatze/alertmanager-bot)
+* [Audio Text-to-Speech](https://www.robustperception.io/audio-alerting-with-prometheus/)
```

Comment:
I don't think you should list this, it's a very silly example for a rather old version of the AM